solidity/dashboard/src/components/AuthorizeContracts.jsx

```diff
@@ -65,41 +66,27 @@ const AuthorizeContracts = ({
           header="operator contract details"
           field=""
           renderContent={({ contracts, operatorAddress }) => (
-            <Contracts
-              contracts={contracts}
-              operatorAddress={operatorAddress}
-              onAuthorizeBtn={onAuthorizeBtn}
-              onDeauthorizeBtn={onDeauthorizeBtn}
-              onSuccessCallback={onSuccessCallback}
-            />
+            <ul className="line-separator">
+              {contracts.map((contract) => (
+                <AuthorizeContractItem
+                  key={contract.contractName}
+                  {...contract}
+                  operatorAddress={operatorAddress}
+                  onAuthorizeBtn={onAuthorizeBtn}
+                  onDeauthorizeBtn={onDeauthorizeBtn}
+                  onSuccessCallback={onSuccessCallback}
+                />
+              ))}
+            </ul>
```

Comment:
Not sure why we are getting rid of `Contracts` component here. Both solutions are valid but i think the best practise is to separate everything as much as possible into separate components. Who knows, maybe we would want to reuse it in the future?

Reply:
I decided to remove `Contracts` due to the prop-drilling. But yeah it makes sense to have it in a separate component, I'll update.